PyISY/Nodes/node.py

```diff
@@ -157,22 +157,8 @@ def dim(self):
             return True
 
     def _get_notes(self):
-        # Get the device notes, currently only the Spoken property is saved.
```

Comment:
Got it.  That answered my first question... Should have read all comments before commenting.